int32 battery_1
int32 battery_2
int32 battery_3
int32 battery_4
int32 battery_5
